lib/cc/analyzer/container.rb

```diff
@@ -75,13 +75,8 @@ def run(options = [])
         )
       ensure
         t_timeout.kill if t_timeout
```

Comment:
:lipstick:, but looking at this now I think it would be nicer to read as `[t_timeout, t_out, t_err].each { |t| t.kill if t }`